### DISCUSSION TITLE
How can I highlight just a image?

### DISCUSSION THREAD

**Discussion by italovieira** [omnivore-app/omnivore#3831] [Q&A] [answered]
I was able to highlight by selecting the text above and  then the image and then the text below the image.
But sometimes I want to highlight just a image from an article without text, but there's no clear way to do that.

**Comment by jacksonh** [comment, marked as answer]
Yeah currently there isn't a way to highlight just an image, other than selecting around it. We could do a highlight button on the images, but tapping them already opens the lightbox display, so we couldn't do it on tap probably

**Reply by italovieira** [reply]
I'm not a UX specialist, but I thougth about using click-and-hold on a image could open the highlight modal.
